tracing-error/src/lib.rs

```diff
@@ -117,7 +117,84 @@
     while_true
 )]
 mod backtrace;
+#[cfg(not(feature = "stack-error"))]
+mod heap_error;
 mod layer;
+#[cfg(feature = "stack-error")]
```

Comment:
Can we add a quick mention of the existence of this feature flag in the main lib docs?